### PR TITLE
Runnable and improved intro examples

### LINKED ISSUE #164: Examples in introduction are not runnable
E.g. missing `action_type=`.

### PR DESCRIPTION
**Pull request by itamarst**
Fixes #164.

### CODE REVIEW

File: docs/source/introduction.rst [hunk 1]
```diff
@@ -64,66 +64,21 @@ The Solution: Eliot
...
-          # Load a Place from the database and return it...
-
-
-  def honeymoon(family):
-      Message.log(message_type="honeymoon",
-                  family=[unicode(person) for person in family])
-      rome = Place.load("Rome, Italy")
-      for person in family:
-          rome.travel(person)
+.. literalinclude:: ../../examples/rometrip_messages.py
```
Comment (wallrj) [inline docs/source/introduction.rst]: :question: It'd be nice to add a download link too.

File: docs/source/introduction.rst [hunk 2]
```diff
@@ -131,91 +86,53 @@ The solution is to introduce two new concepts: actions and tasks. An “action
...
-                      person.look(thing)
-
 
-  def honeymoon(family):
-      with start_task("honeymoon",
-                     family=[unicode(person) for person in family]):
-          rome = Place.load("Rome, Italy")
-          for person in family:
-              rome.travel(person)
+.. literalinclude:: ../../examples/rometrip_actions.py
```
Comment (wallrj) [inline docs/source/introduction.rst]: :question: Download link here too.

**Comment by wallrj** [comment]
Thanks @itamarst 

This looks great. I tried both examples and piped the output through eliot-tree.

I think it might be nice to provide download links to make it even easier to run the examples. Up to you.

Please merge.

**Comment by itamarst** [comment]
Yeah, I opened an issue for that; I'm too tired to actually do it now. Thanks!

**Pull request merged**